image/fetcher.go

```diff
@@ -82,8 +82,9 @@ func (f *Fetcher) pullImage(ctx context.Context, imageID string) error {
 	if err != nil {
 		return err
 	}
-	termFd, isTerm := term.GetFdInfo(f.logger.RawVerboseWriter())
-	err = jsonmessage.DisplayJSONMessagesStream(rc, &colorizedWriter{f.logger.RawVerboseWriter()}, termFd, isTerm, nil)
+	writer := logging.NewWriter(f.logger.Debug)
+	termFd, isTerm := term.GetFdInfo(writer)
```

Comment:
Is this validated empirically? I remember having trouble getting `GetFdInfo` to work on a custom writer.